data/human/hails.txt

```diff
@@ -2089,6 +2091,40 @@ phrase "friendly civilian"
 		"... it's like chess played in space."
 		"..." 3
 
+phrase "friendly civilian"
+	word
+		"If you ever get the chance, you should"
+		"One thing I'd like to do before I kick the bucket is to"
+		"Before you die, you've got to"
+		"For my next trip, I'm going to"
+	word
+		" "
+	word
+		"see the Quarg ringworld in Enif. Isn't it amazing what they can do and we can't?"
+		"see the Quarg ringworld down south. It's such a magnificent feat of engineering."
```

Comment:
The period shouldn't be there and the question mark will need a special replacement.